src/main.js

```diff
@@ -335,6 +335,263 @@ function signLogicSigTransaction(txn, lsig) {
     };
 }
 
+
+/**
+ * makePaymentTxn takes payment arguments and returns a transaction.Transaction object representing that payment txn
```

Comment:
I think a `Transaction` object is sufficient, no need for `transaction.Transaction`